easybuild/easyconfigs/o/olaFlow/olaFlow-20210820-foss-2021b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'Tarball'
+
+name = 'olaFlow'
+version = '20210820'
+local_commit = 'b3039e055a22ea0b8c7dc9f735f15686976559f6'
+
+homepage = 'https://olaflow.github.io/'
+description = """olaFlow CFD Suite is a free and open source project committed to bringing the latest
+advances for the simulation of wave dynamics to the OpenFOAM® and FOAM-extend communities."""
+
+source_urls = ['https://github.com/phicau/olaFlow/archive/']
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'download_filename': '%s.tar.gz' % local_commit,
+}]
+checksums = ['126777c2dc5e8fe563251ce9cc8438bd2e90771cc5a9ea2fc47bd4679502c08b']
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('OpenFOAM', 'v2112')
+]
+
+postinstallcmds = ["source $FOAM_BASH && "
```

Comment:
Using the `Tarball` easyblock with `postinstallcmds` to install something is a bit hackish...

This could be done cleaner using the `Binary` easyblock, and `install_cmd`, see also #10677:

```python
install_cmd = "source $FOAM_BASH && export FOAM_USER_APPBIN=%(installdir)s/bin && "
install_cmd += "export FOAM_USER_LIBBIN=%(installdir)s/lib && /allMake"
```